bentoml/yatai/repository/metadata_store.py

```diff
@@ -251,7 +251,9 @@ def list(
                 # filter_by apply filtering criterion to a copy of the query
                 query = query.filter_by(name=bento_name)
             query = query.filter_by(deleted=False)
-            if label_selectors.match_labels or label_selectors.match_expressions:
+            if label_selectors is not None and (
+                label_selectors.match_labels or label_selectors.match_expressions
+            ):
                 bento_ids = filter_label_query(
```

Comment:
We need to refactor this.. instead of querying by labels, get the ides and query again for Bentos in the ids list, it should be just one query.  filter_label_query function can be refactored to just modify the existing `query` object

Reply:
Can you create an issue for it?

Reply:
https://github.com/bentoml/BentoML/issues/1395